hdf5-vol-daos.spec

```
@@ -197,6 +197,10 @@ done
 %endif
 
 %changelog
+* Wed Jan 27 2021 Maureen Jean <maureen.jean@intel.com> - 1.1.0rc1.g9eebd90
```

No commit hash for tags/releases:
```suggestion
* Wed Jan 27 2021 Maureen Jean <maureen.jean@intel.com> - 1.1.0~rc1
```